native/src/main/java/io/ballerina/stdlib/http/api/logging/HttpLogManager.java

```diff
@@ -161,10 +168,22 @@ public void setHttpAccessLogHandler(BMap accessLogConfig) {
             }
         }
 
+        BString logFormat = accessLogConfig.getStringValue(HTTP_LOG_FORMAT);
+        if (logFormat != null &&
+                !(logFormat.getValue().equals(HTTP_LOG_FORMAT_JSON) ||
+                        logFormat.getValue().equals(HTTP_LOG_FORMAT_FLAT))) {
+            stdErr.println("WARNING: Unsupported log format '" + logFormat.getValue() +
+                    "'. Defaulting to 'flat' format.");
+        }
+
+        BArray logAttributes = accessLogConfig.getArrayValue(HTTP_LOG_ATTRIBUTES);
+        if (logAttributes != null && logAttributes.getLength() == 0) {
```

Comment:
Is there a scenario where `logAttributes` can be null? If so we should have a check for that as well

Reply:
We do check this, if `logAttributes` is null, we consider it as the default case.

Reply:
Check native/src/main/java/io/ballerina/stdlib/http/api/logging/accesslog/HttpAccessLogConfig.java (L77-86)